devnet/substrate_alice.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM parity/polkadot:v0.9.10 AS polkadot
+FROM golang:1.17
+
+ARG CHAIN=cross-client
+ARG VERSION=v0.9.10
+ARG DD_API_KEY=somekey
+ARG METRICS_NAMESPACE=substrate.local.devnet
+
+ENV CHAIN=${CHAIN}
```

Comment:
Should we have the Chain env and arg variables have different names for clarity?

Reply:
Not really, both values are used only once in the same place, docker file uses `ARG CHAIN` only in build time and we need to use the `CHAIN` value in the `ENTRYPOINT` which is only executed after the build time. To achieve this we just assign the `ARG` to `ENV` and now we can access the variable value after build time

Reply:
Yes that makes sense. Ultimately we just want to inject easily the `ENV CHAIN` with `--build-arg CHAIN=blabla`, and that does it 😉 Taking the liberty to resolve this.